### PR TITLE
Fix-up local dev env

### PR DESCRIPTION
**Pull request by ben-challis**

### CODE REVIEW

File: local/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM php:8.2.10-cli-alpine as base
```
Comment (marmichalski) [inline local/Dockerfile:1]: As the project supports 8.1 as minimum, I guess we should use that for local env? Although there is CI in place, so... :shrug:

**Review by marmichalski — DISMISSED** [review]

File: local/Dockerfile (hunk 1 above)
Comment (ben-challis) [reply local/Dockerfile]: No that's fair enough, reverting back to 8.1

File: local/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM php:8.1.23-cli-alpine as base
```
Comment (marmichalski) [inline local/Dockerfile:1]: pls bump https://github.com/Lendable/aggregate/blob/d572f850a909e9a83f0aa8e937622e9574dfeb37/composer.json#L48 :see_no_evil:

**Review by marmichalski — APPROVED** [review]
:trophy:

**Pull request merged**